### ISSUE TITLE
Keyboard based tool selection not working in brave browser

### ISSUE THREAD

**Issue by kishaningithub** [excalidraw/excalidraw#6280]
Keyboard based tool selection not working in brave browser

## Screenshot
<img width="544" alt="image" src="https://user-images.githubusercontent.com/763760/221194613-c4b1b64c-60ba-4cb1-8d53-486f1cebcdef.png">

## Steps to simulate
- Open https://excalidraw.com/ in the latest brave browser
- Press 2 
   - Expected - Square tool should get activated 
   - Actual - Nothing happens
- Press 23
   - Expected - Diamond tool should get activated 
   - Actual - Nothing happens

## Platform
- Mac OSX Ventura running Apple M1 Pro
- Brave browser `Version 1.48.171  Chromium: 110.0.5481.177 (Official Build)  (arm64)`

**Comment by ad1992** [comment]
Hey @kishaningithub I just tried in Brave and its working fine for me. 
I am on Mac as well, brave version - Version 1.48.171 Chromium: 110.0.5481.177 (Official Build) (arm64)

**Comment by kishaningithub** [comment]
@ad1992 What was the tool that you used to record the video? I am still facing this issue though.

**Comment by ad1992** [comment]
Hey, it's ScreenFlow. I think it could be related to `Ventura`

**Comment by kishaningithub** [comment]
Still the same issue exists. I saw the following blocking that brave browser by default does. Could it be because of that?

## Brave browser console log

```
Error with Feature-Policy header: Unrecognized feature: '*'.
excalidraw.com/:1     GET https://cdn.usefathom.com/script.js net::ERR_BLOCKED_BY_CLIENT
excalidraw.com/:1     GET https://scripts.simpleanalyticscdn.com/latest.js net::ERR_BLOCKED_BY_CLIENT
fetch.ts:135     POST https://sentry.io/api/5179260/envelope/?sentry_key=7bfc596a5bf945eda6b660d3015a5460&sentry_version=7 net::ERR_BLOCKED_BY_CLIENT
(anony
```

## Screenshot 

<img width="800" alt="image" src="https://github.com/excalidraw/excalidraw/assets/763760/9b0394f0-4a1d-4eb7-8d1d-1d0360c8f104">

**Comment by dwelle** [comment]
@kishaningithub no, those are just privacy-friendly analytics we're testing & error reporting that Brave is blocking anyway.

So it's only keyboard shortcuts that's affected? Using mouse/touchpad works? Similarly, it works in other browsers?

**Comment by kishaningithub** [comment]
Yes, its only the keyboard shortcuts that is affected. everything else is fine

**Comment by mericano1** [comment]
same issue here

**Comment by kishaningithub** [comment]
Well.  i started using the obsidian excalidraw plugin instead of the website

https://github.com/zsviczian/obsidian-excalidraw-plugin
